index.html

```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+            <li class="profesional-Art-li">bootstrap</li>
+            <li class="profesional-Art-li">Ruby</li>
+          </ul>
+          <a href="#4" class="profesional-Art-btm button">See Project</a>
+        </div>
+        <div class="art-container">
+          <h2 class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </h2>
+          <p class="profesional-Art-content">
+            A daily
+            selection of privately personalized reads;
+            no accounts or sign-ups required. has been the
+            industry's standard
+          </p>
+          <ul class="profesional-Art-ul">
+            <li class="profesional-Art-li">html</li>
+            <li class="profesional-Art-li">bootstrap</li>
+            <li class="profesional-Art-li">Ruby</li>
+          </ul>
+          <a href="#5" class="profesional-Art-btm button">See Project</a>
+        </div>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
+          <p class="profesional-Art-content">
+            A daily
+            selection of privately personalized reads;
+            no accounts or sign-ups required. has been the
+            industry's standard
+          </p>
+          <ul class="profesional-Art-ul">
+            <li class="profesional-Art-li">html</li>
+            <li class="profesional-Art-li">bootstrap</li>
+            <li class="profesional-Art-li">Ruby</li>
+          </ul>
+          <a href="#6" class="profesional-Art-btm button">See Project</a>
+        </div>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
+          <p class="profesional-Art-content">
+            A daily
+            selection of privately personalized reads;
+            no accounts or sign-ups required. has been the
+            industry's standard
+          </p>
+          <ul class="profesional-Art-ul">
+            <li class="profesional-Art-li">html</li>
+            <li class="profesional-Art-li">bootstrap</li>
+            <li class="profesional-Art-li">Ruby</li>
+          </ul>
+          <a href="#7" class="profesional-Art-btm button">See Project</a>
+        </div>
+      </section>
+      <section class="about">
+        <div class="about-me-footer">
+          <div>
+            <h3>About me</h3>
+            <p class="footer-text">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <a href="#" class="footer-btm button">Get My Resume</a>
+          </div>
+          <div class="footer-img">
+            <img src="/Images/illustration-about-me-mobile.png" alt="img">
```

Comment:
- [ ] As mentioned by the previous reviewer kindly use remove this `img` tag and use the `div` tag then give some class name and then use that class name in your CSS file and set your image inside your CSS file :+1:  check my suggestion below :point_down: 

 ```suggestion
            <div class="about-image"></div>
```

**Check below :point_down:  my style suggestions**

```
.about-image{
   background-image: url('./Images/illustration-about-me-mobile.png');
  background-repeat: no-repeat;
}
```